locales/pt_BR/LC_MESSAGES/duckduckgo.po

```diff
@@ -3440,29 +3454,29 @@ msgstr "Selecione %sConfigurações%s."
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSearch Engine%s"
-msgstr ""
+msgstr "Selecione %sMecanismo de Pesquisa%s"
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSearch engine%s, then %sOthers...%s"
-msgstr ""
+msgstr "Selecione %s Mecanismo de Pesquisa %s, em seguida em %sOthers...%s"
 
 msgid "Select %sSettings%s"
 msgstr "Selecione %sConfigurações%s"
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSettings%s"
-msgstr ""
+msgstr "Selecione %sSettings%s"
 
 msgid "Select %sSettings%s from the drop-down menu."
 msgstr "Selecione %sConfigurações%s no menu suspenso."
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSettings%s, then %sAdvanced settings%s"
-msgstr ""
+msgstr "Selecione %sSettings%s, em seguida, %sConfigurações Avançadas%s"
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSettings%s, then %sDefault search engine%s"
-msgstr ""
+msgstr "Selecione %Settings%s, em seguida, %Mecanismo de pesquisa padrão%s"
```

Comment:
The error is here - the tests are failing because the number of `%s` placeholders doesn't match the original.

Easy enough to fix:

```suggestion
msgstr "Selecione %Settings%s, em seguida, %sMecanismo de pesquisa padrão%s"
```

Reply:
Ohhh, Thank you! I'll try that.

Reply:
It worked, thank you. Now I know how to inspect possible CI errors